benchmarks/viscoelastic_sheared_torsion/viscoelastic_sheared_torsion.prm

```diff
@@ -0,0 +1,169 @@
+    set Linear solver tolerance             = 1e-9
+  end
+end
+
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X extent  = 1.0
+    set Y extent  = 1.0
+    set Z extent  = 1.0
+    set X repetitions = 5
+    set Y repetitions = 5
+    set Z repetitions = 5
+    set Box origin X coordinate = -0.5
+    set Box origin Y coordinate = -0.5
+  end
+end
+
+subsection Mesh refinement
+  set Strategy                           = maximum refinement function
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+  subsection Maximum refinement function
+    set Function expression              = 0
+  end
+end
+
+
+# Velocity boundary conditions
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = 0:function, 1:function, 2:function, 3:function, 4:function, 5:function
+  subsection Function
+  # circular rotation plus shearing until t=0.5
+  set Function expression = if(t<0.5, -sin(atan2(y,x))*sqrt(y*y+x*x)*42/360 + 0.3*z, -sin(atan2(y,x))*sqrt(y*y+x*x)*42/360) ;\
+                            cos(atan2(y,x))*sqrt(y*y+x*x)*42/360 ;\
+                            0
+  end
+end
+
+# Formulation classification
+subsection Formulation
+  set Enable elasticity = true
+end
+
+# Gravity model
+subsection Gravity model
+  set Model name  = vertical
+  subsection Vertical
+    set Magnitude = 1
+  end
+end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 7
+  set Names of fields  = ve_stress_xx, ve_stress_yy, ve_stress_zz, ve_stress_xy, ve_stress_xz, ve_stress_yz, goo
+end
+
+# Composition boundary conditions
+subsection Boundary composition model
+  set List of model names = initial composition
+end
+
+# Spatial domain of different compositional fields.
+# The elastic stress tensor components are equal to 0 initially.
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Function expression = 0.0;0.0;0.0;0.0;0.0;0.0;1.0
+  end
+end
+
+# Material model
+subsection Material model
+  set Model name = visco plastic
+
+  subsection Visco Plastic
+
+    set Prefactors for dislocation creep          = 0.005
+    set Stress exponents for dislocation creep    = 1.0
+    set Activation energies for dislocation creep = 0.
+    set Activation volumes for dislocation creep  = 0.
+
+    set Viscous flow law = dislocation
+
+    set Include viscoelasticity = true
+
+    set Densities                   =  1
+
+    set Elastic shear moduli        = 1.e2
+    set Fixed elastic time step     = 0.01
+    set Use fixed elastic time step = false
+    set Use stress averaging        = false
+    set Viscosity averaging scheme  = maximum composition
+
+    set Cohesions                   = 15
+    set Angles of internal friction = 0.
+
+    set Minimum viscosity           = 1e-3
+    set Maximum viscosity           = 1e3
+
+    set Reference viscosity         = 10
```

Comment:
How did you get to this value for the reference viscosity? Is this parameter still needed now that we have the reference viscosity automatically calculated (#4339)?

Reply:
I will check that! (Also for the simple shear BM, from where that line comes)